extensions/rates/functions/src/config.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+  location: process.env.LOCATION!,
+  databaseInstance: process.env.SELECTED_DATABASE_INSTANCE!,
+  shipEngineApiKey: process.env.SHIPENGINE_API_KEY!,
+  collectionPath: process.env.COLLECTION_PATH!,
+  inputKey: process.env.INPUT_KEY!,
+  resultKey: process.env.RESULT_KEY!,
```

Comment:
I don't see either of these in the extensions.yaml – aren't they supposed to be defined there?